WordPress/src/main/java/org/wordpress/android/ui/posts/HistoryListFragment.kt

```diff
@@ -147,9 +147,12 @@ class HistoryListFragment : Fragment() {
             }
         })
 
-        viewModel.showDialog.observe(this, Observer {
-            if (it is Revision && activity is HistoryItemClickInterface) {
-                (activity as HistoryItemClickInterface).onHistoryItemClicked(it, viewModel.revisionsList)
+        viewModel.showDialog.observe(this, Observer { showDialogItem ->
+            if (showDialogItem.historyListItem is Revision && activity is HistoryItemClickInterface) {
```

Comment:
`showDialogItem` is nullable, we might want to consider checking that it's not null or adding requireNoNull. Wdyt?

SideNote:
Since we are touching this code, we might want to consider moving the `activity is HistoryItemClickInterface` check into onAttach and throwing an exception when the activity doesn't implement mentioned interface. Wdyt?